raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -37,6 +38,28 @@ library ChannelManagerLibrary {
     /// @return channel (address) the address of the NettingChannelContract of the two parties.
     function getChannelWith(Data storage self, address caller_address, address partner)
         constant
+        returns (address, bool)
+    {
+        bytes32 party_hash = partyHash(caller_address, partner);
+        if (self.channel_addresses[party_hash] == 0x0 || 
+            !contractExists(self, self.channel_addresses[party_hash])) {
+            return (0x0, false);
+        } else {
+            return (self.channel_addresses[party_hash], true);
+        }
+    }
+
+    /// @notice getChannelWithForDelete(address) to get the address of the unique channel of two parties.
+    /// @dev Get the channel of two parties
+    /// @param caller_address (address) the address of the caller
+    /// @param partner (address) the address of the partner
+    /// @return (address) the address of the NettingChannelContract of the two parties.
+    /// @return (bool) value to reflect if a channel exists
+    /// @return (uint) index of channel in our_channels
+    /// @return (uint) index of channel in partner_channels
+    function getChannelWithForDelete(Data storage self, address caller_address, address partner)
```

Comment:
I strongly suggest changing the name here. May I suggest `getChannelAndIndices()` ?